ansible/library/extract_log.py

```diff
@@ -122,6 +122,7 @@ def extract_number(s):
 def convert_date(fct, s):
     dt = None
     re_result = re.findall(r'^\S{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.?\d*', s)
+    re_result_with_year = re.findall(r'^\d{4}\s{1}\S{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.?\d*', s)
```

Comment:
It would be better to do the second match if `len(re_result) == 0` to reduce time cost.

Reply:
> It would be better to do the second match if `len(re_result) == 0` to reduce time cost.

@bingwang-ms , updated.